src/components/Login.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { setLocalStorage } from '../logics/localStore';
+import { fetchUsers } from '../redux/user/user';
+
+const Login = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMsg, setErrorMsg] = useState('');
+  const store = useSelector((store) => store.userReducer);
+
+  const authenticateUser = (user) => {
+    const error = { error: 'User not found' };
+    try {
+      store.forEach((storeUser) => {
+        if (user.email === storeUser.email && user.password === storeUser.password) {
+          setErrorMsg('');
+          setLocalStorage({ user_id: storeUser.id, loggedIn: true });
+          navigate('/');
+          throw error;
+        } if (user.email !== storeUser.email || user.password !== storeUser.password) {
+          setErrorMsg('Invalid User');
+          throw error;
+        }
+      });
+    } catch (e) {
+      if (e !== error) throw e;
+    }
+    return user;
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    authenticateUser({ email, password });
+  };
+
+  const handleInput = (e) => {
+    const inputField = e.target;
+    const inputValue = inputField.value;
+    if (inputField.name === 'email') {
+      setEmail(inputValue);
+    } else if (inputField.name === 'password') {
+      setPassword(inputValue);
+    }
+    setErrorMsg('');
+    return inputValue;
+  };
+
+  useEffect(() => {
+    if (store.length === 0) {
+      dispatch(fetchUsers());
+    }
+  }, []);
+  return (
+    <section className="authentication-section">
```

Comment:
- [ ] It looks like this part of the project could improve the style, look how the form is not completely visible.
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/20563252/176568797-f817dafe-d3d5-4bc3-980e-108e5ca0e981.png">

- [ ] I got another issue here, once a create my own user, this wasn't saved on the backend, consider seeing this issue too.